resources/css/button.css

```diff
@@ -38,7 +38,8 @@
   background: none;
 }
 .btn-link:hover {
-  border: none;
+  background: none !important;
+  border: none !important;
```

Comment:
`!important` is kind of lame, but this solves the bug with link buttons causing a layout shift on hover.

Reply:
I can't seem to replicate the behaviour where those buttons cause layout shifts without `!important`. Is this happening on a specific page?

Reply:
Yep, check out friends.php

![Screenshot 2023-09-04 at 2 24 51 PM](https://github.com/RetroAchievements/RAWeb/assets/3984985/20ce58bd-3086-4d74-8f4a-f2adcf7deeb6)

These three buttons shift the layout around them when hovered

Reply:
oh, i understand. it's because of the higher specifity of `:enabled`.
adding the border just for , even for `.btn-link` 🤔

Can we remove the `:enabled` selector as it's only applied to `button` elements, override hover for disabled instead? This way it doesn't interfere with the more generic `.btn-link` styles.

Something like this:
```css
.btn:hover {
  background-color: var(--embed-highlight-color);
  color: var(--link-hover-color);
  border: 1px solid var(--menu-link-color);
}
.btn:disabled, .btn:disabled:hover {
  color: var(--text-color-muted);
  border: 1px solid var(--embed-color);
  cursor: not-allowed;
}

.btn-link {
  border: none;
  background: none;
}
.btn-link:hover {
  border: none;
  background: none;
}
```

Reply:
Same issue with this change on my local (even after restarting Vite). Going to play with it for just a minute...

Reply:
Think I got it:

```css
.btn-link {
  border: 1px solid transparent;
  background: none;
}
.btn-link:hover {
  border: none;
}
```

Reply:
with that it starts jumping for me 😅

Reply:
Latest changes in https://github.com/RetroAchievements/RAWeb/pull/1821/commits/0e42a8bae7d3e70dbad56ef3eb183e45c8c80151

Reply:
Looking good, thanks!